src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/controllers/ResponseExceptionHandler.java

```diff
@@ -0,0 +1,23 @@
+package uk.gov.hmcts.reform.bulkscan.orchestrator.controllers;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.servlet.mvc.method.annotation.ResponseEntityExceptionHandler;
+
+import static org.springframework.http.HttpStatus.INTERNAL_SERVER_ERROR;
+import static org.springframework.http.ResponseEntity.status;
+
+@ControllerAdvice
+public class ResponseExceptionHandler extends ResponseEntityExceptionHandler {
+
+    private static final Logger log = LoggerFactory.getLogger(ResponseExceptionHandler.class);
+
+    @ExceptionHandler(Exception.class)
+    protected ResponseEntity<Void> handleInternalException(Exception exception) {
+        log.error(exception.getMessage(), exception);
```

Comment:
Instead of `exception.getMessage()` we could have a message saying that an error occurred when serving a request.